.changeset/orange-roses-give.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": minor
```

Comment:
If we are making the prop required it should be a breaking change; even if it's not breaking dotcom, it might be breaking for other users of Primer React.